cosmwasm/packages/sgx-vm/src/enclave.rs

```diff
@@ -125,3 +163,30 @@ pub fn configure_enclave(config: EnclaveRuntimeConfig) -> SgxResult<()> {
 
     Ok(())
 }
+
+/// This is a safe wrapper for allocating buffers inside the enclave.
+///
+/// It must be called after the enclave has been initialized, and can not be called
+/// while another thread is using the enclave, or it will panic.
+pub(super) fn allocate_enclave_buffer(buffer: &[u8]) -> SgxResult<EnclaveBuffer> {
```

Comment:
I moved this function here just to clean up the code a little. It was in a really inappropriate location before